src/app/chart-docs/chart-types-docs/line-docs/line-docs.component.html

```diff
@@ -0,0 +1,136 @@
+<mat-card flex>
+  <div layout="row" layout="center center">
+    <div flex>
+        <mat-card-title>Line Test (atomic)</mat-card-title>
+        <mat-card-subtitle>Individual component configurations, series, tooltip, x & y axis, series tooltip</mat-card-subtitle>
+    </div>
+    <div layout="column" layout-align="center center" class="push-right-sm">
+        <button class="text-upper" mat-button (click)="showTooltip = !showTooltip">
```

Comment:
Would be nice if this were a toggle or something that indicates the current state (show/hidden)